rsync-homedir-excludes.txt

```diff
@@ -269,3 +270,9 @@ Thumbs.db
 
 # other temporary files 
 nohup.out
+
+# LibreOffice cache files
+.config/libreoffice/4/cache
+
+# freshplayerplugin LSO
+.config/freshwrapper-data/Shockwave Flash/WritableRoot/#SharedObjects
```

Comment:
did you test this with the unescaped space in Shockwave Flash? maybe it has to be` Shockwave\ Flash` ?

Reply:
I tested it. Neither the space nor `#` needs to be escaped.